apps/test-can/new-can-rx-test/readme.txt

```diff
@@ -0,0 +1,11 @@
+#######
+I've implemented a new method for processing incoming CAN msgs.
+
+The workflow should go like this:
+1. CAN Msg arrives
+2. interrupt is trigerred
+3. the postman thread puts reads the msg and puts in the mailbox
```

Comment:
lol im getting PTSD from looking at this stuff. but yeah this readme is garbage and shouldnt be in master

Reply:
Neopixel 😈